services/github/github-lerna.service.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { renderVersionBadge } = require('../version')
+const { semver } = require('../validators')
+const { ConditionalGithubAuthV3Service } = require('./github-auth-service')
+const { fetchJsonFromRepo } = require('./github-common-fetch')
+const { documentation } = require('./github-helpers')
+
+const versionSchema = Joi.object({
+  version: semver,
+}).required()
+
+module.exports = class Lerna extends ConditionalGithubAuthV3Service {
+  static get category() {
+    return 'version'
+  }
+
+  static get route() {
+    return {
+      base: 'github/lerna',
+      pattern: ':user/:repo/:branch*',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Github lerna version',
+        pattern: ':user/:repo',
+        namedParams: { user: 'babel', repo: 'babel' },
+        staticPreview: this.render({ version: '7.6.4' }),
+        documentation,
+      },
+      {
+        title: 'Github lerna version (branch)',
+        pattern: ':user/:repo/:branch',
+        namedParams: { user: 'jneander', repo: 'jneander', branch: 'colors' },
+        staticPreview: this.render({
+          version: 'independent',
+          branch: 'colors',
+        }),
+        documentation,
+      },
+    ]
+  }
+
+  static render({ version, branch }) {
+    return renderVersionBadge({
+      version,
+      tag: branch,
+      defaultLabel: 'lerna',
```

Comment:
It'd be better to extract the label out into the `defaultBadgeData` getter function. This way the label will be set even in the event of errors, such as 404/not found scenarios

```js
  static get defaultBadgeData() {
    return {
      label: 'lerna'
    }
  }
```

Reply:
Actually there's a specific reason why it's done like this. If you extracted to `defaultBadgeData`, badges that don't specify a tag (i.e. branch in this case) would become `lerna | v1.0.0`. If you just specify as `defaultLabel`, they remain `version | v1.0.0`, and only tagged ones will contain the Lerna keyword, e.g.  `lerna@master | v1.0.0`.

Not sure whether it's a good reason though, but it's consistent with the behaviour in our _manifest.json_ and _package.json_ badges.

Reply:
> Not sure whether it's a good reason though, but at least it's consistent with the behaviour in our manifest.json and package.json badges.

True for manifest.json, but isn't the label for package.json always `version` even when the user specifies a branch? And for another flavor, the go.mod version badges from GH repos are `go | v1.0.0` and `go@master | v1.0.0`. 🤷‍♂ 

I'm not sure which of those patterns the lerna badge should follow 🤔 Should it mimic the package.json pattern or do we want to have `lerna` explicitly in the label?

Reply:
Changed. Shall we do something about our _manifest.json_ and _package.json_ badges in a separate issue? I've got a feeling _package_ and _manifest_ are very overloaded terms, so I would suggest simply sticking with `version` everywhere for those. Or should we just ignore the inconsistency?

Reply:
> Shall we do something about our manifest.json and package.json badges in a separate issue? I've got a feeling package and manifest are very overloaded terms, so I would suggest simply sticking with version everywhere for those. Or should we just ignore the inconsistency?

I think it's worth reviewing and potentially updating yes. `version` makes sense to me for the package.json, but I don't even remember offhand what `manifest` is used for